pyproject.toml

```diff
@@ -59,7 +59,7 @@ mockupdb = [
     "mockupdb@git+https://github.com/mongodb-labs/mongo-mockup-db@master"
 ]
 pymongocrypt_source = [
-    "pymongocrypt@git+https://github.com/mongodb/libmongocrypt@master#subdirectory=bindings/python"
+    "pymongocrypt@git+https://github.com/ShaneHarvey/libmongocrypt@PYTHON-5191#subdirectory=bindings/python"
```

Comment:
was this an intentional change?

Reply:
Yes, I needed this to test the new changes. Now that https://github.com/mongodb/libmongocrypt/pull/984 is merged I was able to revert this.